src/lib/storage/boot_requirements_strategies/base.rb

```diff
@@ -31,9 +38,14 @@ module BootRequirementsStrategies
       class Base
         include Yast::Logger
 
+        using Refinements::Disk
+        using Refinements::DevicegraphLists
+
         def initialize(settings, disk_analyzer)
           @settings = settings
           @disk_analyzer = disk_analyzer
+          @root_disk = @disk_analyzer.devicegraph.disks.with(name: settings.root_device).first
```

Comment:
Hmmm, AFAIK one of the goals of the `DiskAnalyzer` class was to hide the existence and details of `Devicegraph` (you know, "we are not in the business of doing devicegraphs")[1]. Not sure if `@root_disk` is a consistent idea with the original design.

[1] From that point of view, maybe even fact that `DiskAnalyzer#devicegraph` is public can be considered kind of a bug.